test/e2e/suite/command/cp.go

```diff
@@ -80,6 +79,15 @@ var _ = Describe("1.1 registry users:", func() {
 			CompareRef(src, dst)
 		})
 
+		It("should copy and show debug logs with duplication", func() {
```

Comment:
nit: without duplication? Or something like "should copy and track request number per target"?

Reply:
Trying to say deduplication here, rephrased.